parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -81,6 +82,20 @@
 
   private static final Log LOG = Log.getLog(ParquetMetadataConverter.class);
 
+  private final boolean useSignedStringMinMax;
+
+  public ParquetMetadataConverter() {
+    this(false);
+  }
+
+  public ParquetMetadataConverter(Configuration conf) {
+    this(conf.getBoolean("parquet.strings.signed-min-max.enabled", false));
```

Comment:
We removed the public constant to avoid leaking this into the public API and not being able to remove it. Multiple packages reference it, so it would need to be public. I think it's fine to use a String for now.